src/Microsoft.Identity.Web/WebApiServiceCollectionExtensions.cs

```diff
@@ -124,15 +124,19 @@ public static class WebApiServiceCollectionExtensions
             {
                 options.Events ??= new JwtBearerEvents();
 
+                var onTokenValidatedHandler = options.Events.OnTokenValidated;
+
                 options.Events.OnTokenValidated = async context =>
```

Comment:
@jmprieur Related to your suggestion we can just `+=` our handler to the existing ones, right?
```suggestion
                options.Events ??= new JwtBearerEvents();
                
                options.Events.OnTokenValidated += async context =>
```

Reply:
possibly but the convention within the rest of the library is to do it as I originally did it (in fact I have used exactly the same variable names to be consistent with the "house style".  e.g: https://github.com/AzureAD/microsoft-identity-web/blob/d8608bb000b0d9415a290f5dde836248b179ae38/src/Microsoft.Identity.Web/WebAppServiceCollectionExtensions.cs#L175)

Possibly the += syntax requires null checks or similar?  Also in the *MiddlewareDiagnostics classes += is not used.  So I (personally) would stick with "house style" (or change it everywhere if += truly better).

Reply:
I agree with @jg11jg, we should keep things consistent. I would leave as-is and we can update things later if we want.